code/__HELPERS/visual_effects.dm

```diff
@@ -0,0 +1,23 @@
+/// Makes this atom look like a "hologram"
+/// So transparent, blue, with a scanline and an emissive glow
+/// This is acomplished using a combination of filters and render steps/overlays
+/// The degree of the opacity is optional, based off the opacity arg (0 -> 1)
+/atom/proc/makeHologram(opacity = 0.5, modify_color_transparency = TRUE)
+	// First, we'll make things blue (roughly) and sorta transparent
+	if(modify_color_transparency)
```

Comment:
Is there a particular reason you didnt port the lastest makehologram version?

Reply:
i did, but you'll notice at https://github.com/tgstation/TerraGov-Marine-Corps/pull/17675/commits/5c6b69de1982e0565f898e64cffdb84597a3efcd that it caused a linter error and I couldn't figure out a way to fix it, so I just removed the emissives part, if you have a idea on how to fix that it'd be nice

Reply:
now uses latest TG with one edit on line 7 so it doesn't change color